packages/itwinui-react/src/core/Avatar/Avatar.tsx

```diff
@@ -104,6 +104,7 @@ export const Avatar = React.forwardRef((props, ref) => {
       {status && (
         <Box
           as='span'
+          role='status'
```

Comment:
`role=status` creates a [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions), which is meant for notifications and other updates, so it is the wrong role in this case.

I think there is no special role needed here, and that we should actually remove the aria-label.

We could add the status text as a separate [visually-hidden](https://itwinui.bentley.com/docs/visuallyhidden) span in a future PR if desired.

Reply:
Done